### ISSUE TITLE
Skip checkpoint eval due to throttle secs (18000 secs)

### ISSUE THREAD

**Issue by mohammedayub44** [OpenNMT/OpenNMT-tf#243]
Hi,

While training the models I see that its skipping evaluation for checkpoints, any reason why it's doing that ?

**INFO:tensorflow:Skip the current checkpoint eval due to throttle secs (18000 secs).**
INFO:tensorflow:global_step/sec: 4.23382
INFO:tensorflow:loss = 1.598122, step = 45000 (1180.966 sec)
INFO:tensorflow:words_per_sec/features: 12490.9
INFO:tensorflow:words_per_sec/labels: 12917.7
INFO:tensorflow:Saving checkpoints for 50000 into mayub_try/NMTS_token_bpe_run10252018/model.ckpt.
**INFO:tensorflow:Skip the current checkpoint eval due to throttle secs (18000 secs).**
INFO:tensorflow:global_step/sec: 4.23292
INFO:tensorflow:loss = 1.6568156, step = 50000 (1181.219 sec)
INFO:tensorflow:words_per_sec/features: 12490
INFO:tensorflow:words_per_sec/labels: 12915
INFO:tensorflow:Saving checkpoints for 55000 into mayub_try/NMTS_token_bpe_run10252018/model.ckpt.
**INFO:tensorflow:Skip the current checkpoint eval due to throttle secs (18000 secs).**

Thanks !

Mohammed Ayub

**Comment by guillaumekln** [comment]
Hi,

It just means that a checkpoint was produced but will not be evaluated because `eval_delay` seconds did not pass since the last evaluation or start of the training. It defaults to 5 hours.

**Comment by mohammedayub44** [comment]
@guillaumekln  Thanks.  With the config I have my checkpoints are getting produced every 30 minutes. 
So if I change `eval_delay` to 1500 secs I should see all my evaluation for checkpoints. ?

Mohammed Ayub

**Comment by guillaumekln** [comment]
Yes, the condition to run evaluation is more generally: `eval_delay` seconds has passed and the most recent checkpoint was not already evaluated.
